### PR TITLE
Judges queue | Assign x cases | Attorney side bar

### PR DESCRIPTION
**Pull request by evankroske**
Resolves https://github.com/department-of-veterans-affairs/caseflow/issues/4763

### Description
Adds a sidebar to the judges assign queue that displays the attorneys on their team.

screen shot 2018-05-03 at 10 33 22 am

### Acceptance Criteria 
- [ ] The sidebar includes the attorneys on the viewer's team.

### Testing Plan
- [x] Log in as BVAAABSHIRE.
- [x] On http://localhost:3000/queue/ , click "Switch to Assign Cases".
- [x] The sidebar includes all attorneys that appear on the page http://localhost:3000/users?role=Attorney&judge_css_id=BVAAABSHIRE.

### CODE REVIEW

File: client/app/queue/JudgeAssignTaskListView.jsx [hunk 1]
```diff
@@ -37,28 +36,72 @@ class JudgeAssignTaskListView extends React.PureComponent {
     this.props.resetErrorMessages();
   };
 
-  constructor(props) {
-    super(props);
-    this.state = DISPLAYING_ASSIGN_TASKS;
+  title = (reviewableCount) => <h1>Assign {reviewableCount} Cases</h1>
+  switchLink = (that) => <Link to={`/${that.props.userId}/review`}>Switch to Review Cases</Link>
```
Comment (dannysellers) [inline client/app/queue/JudgeAssignTaskListView.jsx]: This should now start with `/queue` (#5341).

Also, you don't need to pass `this` to functions declared inside the class.

File: client/app/queue/JudgeAssignTaskListView.jsx [hunk 2]
```diff
@@ -37,28 +36,72 @@ class JudgeAssignTaskListView extends React.PureComponent {
...
+  visibleTasks = (tasks) => _.filter(tasks, (task) => task.attributes.task_type === 'Assign')
+  noTasksMessage = () => 'Congratulations! You don\'t have any cases to assign.'
+  table = () => <JudgeAssignTaskTable />
```
Comment (dannysellers) [inline client/app/queue/JudgeAssignTaskListView.jsx]: What's the advantage of making these functions? Outside `visibleTasks` there's not much business logic, and each of the bottom three are only used once

File: client/app/queue/JudgeAssignTaskListView.jsx [hunk 3]
```diff
@@ -37,28 +36,72 @@ class JudgeAssignTaskListView extends React.PureComponent {
...
+    const url = `/users?role=Attorney&judge_css_id=${this.props.userCssId}`;
+
+    return ApiUtil.get(url, requestOptions).
+      then(
+        (response) => {
+          const resp = JSON.parse(response.text);
+
+          this.props.setAttorneysOfJudge(resp.attorneys);
+        },
+        () => null);
```
Comment (dannysellers) [inline client/app/queue/JudgeAssignTaskListView.jsx]: For a future PR: should we display an error on failure?

File: client/app/queue/JudgeAssignTaskListView.jsx (hunk 1 above)
Comment (evankroske) [reply client/app/queue/JudgeAssignTaskListView.jsx]: ✅

File: client/app/queue/JudgeAssignTaskListView.jsx (hunk 3 above)
Comment (evankroske) [reply client/app/queue/JudgeAssignTaskListView.jsx]: LoadingDataDisplay does that for us

File: client/app/queue/JudgeAssignTaskListView.jsx (hunk 2 above)
Comment (evankroske) [reply client/app/queue/JudgeAssignTaskListView.jsx]: ✅

File: client/app/queue/JudgeAssignTaskListView.jsx (hunk 3 above)
Comment (dannysellers) [reply client/app/queue/JudgeAssignTaskListView.jsx]: Oh, right, we have `failStatusMessageChildren`. Cool 👍 

For the future: I wonder if it'd be better to have an error message that specifically mentions what we were loading, something like "Failed to load attorneys". Just saying "sidebar" doesn't give any indication of what the user's missing out on / what exactly failed to load / how it'll affect their workflow

**Pull request merged**